static/js/tippyjs.js

```diff
@@ -245,4 +245,28 @@ export function initialize() {
         },
         appendTo: () => document.body,
     });
+
+    delegate("body", {
+        target: ".message_inline_image > a > img",
+        appendTo: () => document.body,
+        // Add a short delay so the user can mouseover several inline images without
+        // tooltips showing and hiding rapidly
+        delay: [300, 20],
+        onShow(instance) {
+            // Some message_inline_images aren't actually images with a title,
+            // for example youtube videos, so we default to a generic title
+            this.title = $(instance.reference).parent().attr("title") || "attachment";
+            const prefix = $t({defaultMessage: "View or download"});
+            const suffix = " " + this.title;
```

Comment:
This string construction doesn't work with i18n; 
https://zulip.readthedocs.io/en/latest/translating/internationalization.html provides the background but basically different languages have different word orders. So one always wants to pass to the i18n library something like `"View or download {title}", {title: this.title}`, so that translators can change the word order of the string.

Reply:
And I guess you'll want to do something slightly fancier to get the whitespace and `strong` tags to work -- my proposal would be actually to do a tiny HTML template, `static/templates/message_inline_image_tooltip.hbs`.  `static/templates/markdown_time_tooltip.hbs` may be a useful example.

Reply:
I had the thought of using a template once I had to make that second jQuery chain. Good call.